src/form-elements/FormElementSection.tsx

```diff
@@ -33,16 +34,15 @@ function FormElementSection<T extends FormTypes._NestedElementsElement>({
   const displayValidationMessage =
     displayValidationMessages || isDisplayingError
 
-  const isInvalid = React.useMemo(() => {
-    return (
-      displayValidationMessage &&
-      checkSectionValidity(element, props.formElementsValidation)
-    )
-  }, [displayValidationMessage, element, props.formElementsValidation])
-
-  const isValid = React.useMemo(() => {
-    return !checkSectionValidity(element, props.formElementsValidation)
-  }, [element, props.formElementsValidation])
```

Comment:
I would leave this hook here and pass `isValid` as `formElementsValid` as the `checkSectionValidity()` could contain multiple iterations, it will benefit from being memo'd.

Reply:
Good idea